www/netj00/ZP2/data/JS/memeMaker.js

```diff
@@ -0,0 +1,316 @@
+const canvas = document.getElementById("canvas");
+const templates = document.getElementById("templates");
+const text1 = document.getElementById("textinput1");
+const text2 = document.getElementById("textinput2");
+const text3 = document.getElementById("textinput3");
+const color1 = document.getElementById("textColor1");
+const color2 = document.getElementById("textColor2");
+const color3 = document.getElementById("textColor3");
+const color4 = document.getElementById("textColor4");
+const color5 = document.getElementById("textColor5");
+const color6 = document.getElementById("textColor6");
+
+
+
+
+_canvasState = new CanvasState(canvas);
+
+let xmlDoc = null;
+
+function onLoad() {
+    canvas._width = canvas.width;
```

Comment:
dom element ma property `_width`? co takhle to vyresit pres CSS?

Reply:
Používám pro původní rozměry obrázku. Canvas se pak upravuje podle výšky a šířky načteného obrázku.